xbmc/interfaces/swig/AddonModuleXbmcgui.i

```diff
@@ -43,6 +43,8 @@ using namespace xbmcgui;
 // not part of what swig parses.
 %feature("knownbasetypes") XBMCAddon::xbmcgui "AddonClass,AddonCallback"
 
+%feature("knownapitypes") XBMCAddon::xbmcgui "XBMCAddon::xbmc::InfoTagVideo,xbmc::InfoTagMusic"
```

Comment:
I remember now. There's weird bug in the generator I believe, that I could never resolve. It refuse to recognize the type if the XBMCAddon namespace is in the second parameter. Possibly it inherits the top namespace while parsing it or something?